### ISSUE TITLE
[Bug]: Enemies do increasing amounts of damage for every time someone reconnects to session.

### ISSUE THREAD

**Issue by LeDemmers** [LukeYui/EldenRingSeamlessCoopRelease#400]
### Describe the issue in detail

Played with a group of 4, one person disconnected, enemies still did roughly the same amount of damage. The 4th rejoined, and the enemies did 2x the amount of damage and were much more tanky. Two others disconnected, came back and reconnected. Enemies are nigh unkillable, even rats can one-shot any one of us, and even doing 150 damage with a heavy weapon does nothing to the rat. 

TL;DR : Scaling/balance is broken.

### Steps to recreate the bug

Have 3 others join session, disconnect, then reconnect as needed. Each time this happens the scaling increases further until game is unplayable.
Not running any other mods, just base game and the Seamless Mod.

### Version

1.2.5 (Latest)

**Comment by LeDemmers** [comment]
We did not see any buff/debuff on the screen. And this was at the beginning of a session, nothing had been done other than the others joining the game.

**Comment by loki2236** [comment]
I've seen this issue happening only a FEW times. 
If you can @LeDemmers use CheatEngine with the TGA table (maintained by Inunorii, you can find it on github) to check the "NG Cycle" (Like if the games register that you are on journey 2 or 3 everytime someone new connects. (It shouldnt increase, at all, but I've seen it happen)

Its a pretty weird bug and i've only seen it happen twice. So it may not be related to your current bug. But try, and then comment here again.
